lib/child_process.js

```diff
@@ -297,12 +297,14 @@ exports.execFile = function(file /*, args, options, callback*/) {
   return child;
 };
 
-var _deprecatedCustomFds = internalUtil.deprecate(function(options) {
-  options.stdio = options.customFds.map(function(fd) {
-    return fd === -1 ? 'pipe' : fd;
-  });
-}, 'child_process: options.customFds option is deprecated. ' +
-   'Use options.stdio instead.');
+const _deprecatedCustomFds = internalUtil.deprecate(
+  function deprecateCustomFds(options) {
+    options.stdio = options.customFds.map(function(fd) {
```

Comment:
I was actually thinking you were talking about changing

```js
function deprecateCustomFds(options) {
```

To a const, so i misunderstood you. totally missed the call to map.

```js
const mapCustomFds = (fd) => { return fd === -1 ? 'pipe' : fd; };
const _deprecatedCustomFds = internalUtil.deprecate(
  function deprecateCustomFds(options) {
    options.stdio = options.customFds.map(mapCustomFds);
  }, 'child_process: options.customFds option is deprecated. ' +
     'Use options.stdio instead.'
);
```

I wrapped the block for the arrow function in curlys to avoid lint error for no-confusing-arrow. Does this solution look okay to you?